app/views/projects/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class='container'>
+  <div class='row'>
+    <div class='col'>
+      <table class='table mb-5'>
+        <%= link_to 'Create Project', new_user_project_path, class:'btn btn-success mb-2 float-right text-light' %>
+        <thead class='bg-dark text-white'>
+          <tr>
+            <th scope='col'>Title</th>
+            <th scope='col'>Description</th>
+            <th scope='col'>Active</th>
+            <th scope='col'>Start Date</th>
+          </tr>
+        </thead>
+        <% @projects.each do |project| %>
+          <tbody>
+            <tr>
+              <td><%= project.title %></td>
+              <% if project.description.present? %>
+                <td><%= project.description.truncate(28) %></td>  
```

Comment:
make this one liner and all other on this file.
`<%= project.description.truncate(28) if project.description.present? %>`

Reply:
done